src/pages/OrderListPage/OrderListPage.tsx

```diff
@@ -0,0 +1,26 @@
+import useGetOrderList from '../../hooks/requests/useGetOrderList.ts';
+import OrderList from '../../components/@common/OrderList/OrderList.tsx';
+import * as Styled from './OrderListPage.styles.tsx';
+import EmptyComponent from '../../components/@common/EmptyComponent/EmptyComponent.tsx';
+
+const OrderListPage = () => {
+  const { data: orderListData } = useGetOrderList();
+
+  return (
+    <Styled.OrderListPageWrapper>
+      <Styled.OrderListPageContent>
+        <Styled.OrderListPageTitle>주문목록</Styled.OrderListPageTitle>
+        <Styled.OrderListPageTitleBorder />
+        {orderListData && orderListData.orders.length === 0 && <EmptyComponent />}
+
+        {orderListData &&
+          orderListData.orders.length > 0 &&
+          orderListData.orders.map((orderData) => {
+            return <OrderList key={orderData.orderId} orderData={orderData} hasDetailNavigateButton />;
+          })}
```

Comment:
```suggestion
        {orderListData && orderListData.orders.length > 0 ? 
          orderListData.orders.map((orderData) => {
            return <OrderList key={orderData.orderId} orderData={orderData} hasDetailNavigateButton />;
          })}
          : <EmptyComponent />
        }
```

요런 방식은 어떤가요? 하나의 조건식에 내용을 포함할 수 있어서 가독성이 조금 더 좋아질 것 같아요! 🙌

Reply:
[Use ternaries rather than && in JSX by Kent C. Dodds](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)

react conditional rendering 방법중 && 와 삼항 연산자에 대해 항상 고민했었는데, 그루밍 덕분에 학습을 할 수 있었습니다..! 

추천해주신대로 아래와 같이 삼항 연산자를 적용해보았습니다!

### OrderListPage.tsx

```ts
const OrderListPage = () => {
  const { data: orderListData } = useGetOrderList();

  return (
    <Styled.OrderListPageWrapper>
      <Styled.OrderListPageContent>
        <MainPageText>주문목록</MainPageText>
        <MainTextBorder />
        {orderListData && orderListData.orders.length > 0 ? (
          orderListData.orders.map((orderData) => {
            return <OrderList key={orderData.orderId} orderData={orderData} hasDetailNavigateButton />;
          })
        ) : (
          <EmptyCartComponent />
        )}
      </Styled.OrderListPageContent>
    </Styled.OrderListPageWrapper>
  );
};
```